### DISCUSSION TITLE
Error in Jupyter Notebook Example - Client object has no method "get_historical_features"

### DISCUSSION THREAD

**Discussion by rylativity** [feast-dev/feast#1352] [Q&A] [answered]
Hello,

I only recently stumbled across the Feast project, and I think it is very promising for my use case.  I am attempting to learn how Feast works by going through the QuickStart guide.

After starting up the docker environment, I am running into an issue in the [example Jupyter Notebook](https://github.com/feast-dev/feast/blob/master/examples/minimal/minimal_ride_hailing.ipynb). In code-cell 19 of the notebook, there is a method call that is returning the error - Client' object has no attribute 'get_historical_features':

```# get_historical_features will return immediately once the Spark job has been submitted succesfully.
job = client.get_historical_features(
    feature_refs=[
        "driver_statistics:avg_daily_trips",
        "driver_statistics:conv_rate",
        "driver_statistics:acc_rate",
        "driver_trips:trips_today"
    ], 
    entity_source=entities_with_timestamp
)
```

I followed all the instructions to the letter up until that point, and I am receiving the same error on multiple machines.  I also cannot find the method "get_historical_features" defined anywhere in the Feast codebase.  Is there something I am missing?

Thanks in advance!

**Comment by woop** [comment, marked as answer]
Hi @rylativity,

I think there was a problem with the image build. Can you please pull the image again and see if it's fixed? (I submitted a patch last night)

**Reply by rylativity** [reply]
Hi @woop,

Pulling the updated image resolved the issue.  Thank you!
